packages/java-jaxrs-common/templates/pojoContents.hbs

```diff
@@ -212,6 +212,7 @@ public java.lang.String toString() {
 	return sb.toString();
 }
 
+{{#or parentNativeType properties}}
```

Comment:
It appears that we also use `toIndentedString` on line 208 if `additionalProperties` is set, so I think that needs to be added to this condition.

So it seems that the only time we _don't_ use `toIndentedString` is if the model is empty.

Reply:
Whoops. 😬

Reply:
I've amended the PR. I added the `additionalProperties` field to the `{{or}}`, but I wonder if there's some other way of suggesting that the method would be needed. There's no way to set state in handlebars, is there?